assets/js/base/components/cart-checkout/totals/coupon/style.scss

```diff
@@ -39,3 +39,9 @@
 	flex-direction: column;
 	position: relative;
 }
+a.wc-block-components-totals-coupon-link {
+	text-decoration: none !important;
```

Comment:
If I remove important, it won't work for the storefront theme. Is there a better way to tackle this?

Reply:
What was the reason this class was added, cause it might not be needed? That anchor should inherit the theme styles, just like [here](https://github.com/woocommerce/woocommerce-blocks/issues/8027#issuecomment-1376753360), and should work across the themes. For example, in TT2 the anchors are underlined so the line above would prevent the underline.

Reply:
@ralucaStan Yes, we should follow the theme style. However, since it is a link, if I hover over it removes the underline. Here is the example without CSS:

https://user-images.githubusercontent.com/11503784/211550334-edcf2791-5340-451a-a454-50b0648923d6.mov

Reply:
👍  sounds good.